data/hai/hai reveal 4 middle.txt

```diff
@@ -1429,16 +1549,29 @@ mission "Hai Reveal [A11] Sound the Alarm"
 			`	You're still finishing up a little shortly after they leave, at which point a harried-looking Sayari appears and practically sprints across the room to you.`
 				goto alarm
 			label dinner
+			action
+				set "learned of elders superstition"
 			`	You join Teeneep and Sayari at a fusion restaurant about a block down the road from the 'Elders' Chambers' - which turns out to be the official name of the whole government complex.`
-			`	Having finished your main meal, and while waiting for some dessert that Teeneep has ordered, Sayari pulls out her datapad. "You know, while we're waiting. It's probably a good idea to check the damage. I don't know if your plan is actually going to have the desired effect, you know. I mean, seriously, Tee, what will you do if it doesn't? Will the Hai actually have to shed our brethren's blood with lethal intent?"`
-			`	Teeneep just sips her drink and shakes her head. "Oh, Sayari, you really are getting too long in the tooth for domestic politics these days."`
+			`	Having finished your main meal, and while waiting for some dessert that Teeneep has ordered, Sayari pulls out her datapad. "You know, while we're waiting. It's probably a good idea to check the damage. I don't know if your plan is actually going to have the desired effect, you know. Especially after that scene in there; no doubt the long tooths will get riled up."`
+			choice
+				`	"What was that in there about the Mythos?"`
+				`	"What will happen if they get riled up?"`
+					goto riled
+			`	Teeneep rolls her eyes, putting her drink down with a sharp clink. "It's superstition. Humans have their religions, the Hai have the Mythos. It's a collection of stories about Hai who lived incredibly long lives and were involved in or responsible for incredibly influential events in our history, or more accurately pre-history for the vast majority of them. If you believe the stories literally these Hai transcended mortal limitations and achieved immortality; our word for them translates imprecisely to either 'immortals' or 'imperishables'. When someone embodies a character from it well enough there’s a set of people who will believe it’s actually that person come back to steer things.`
```

Comment:
```suggestion
			`	Teeneep rolls her eyes, putting her drink down with a sharp clink. "It's superstition. Humans have their religions, the Hai have the Mythos. It's a bunch of tales about Hai who lived incredibly long lives and were involved in or responsible for incredibly influential events in our history, or more accurately pre-history, for the vast majority of them. If you believe the stories literally, these Hai transcended mortal limitations and achieved immortality; our word for them translates imprecisely to either 'immortals' or 'imperishables'. When someone embodies a character from it well enough, there’s a set of people who will believe it’s actually that person who has come back to steer things.`
```
Imperishable foodstuffs? 

I suppose Teeneep doesn't give a damn for this stuff so I want to make it more clear. I don't remember if 'whose' can be a replacement for 'who has' but it might be like 'might of' and 'might have'. It might not even need to be there... Bloody English.

Reply:
I mean, "who has" is entirely superfluous here, why do we need to add it?

Also, there's a difference between "not giving a damn" and actively denigrating something. "A bunch of tales" is the latter.

Reply:
Well, if I didn't give a damn about religion, I'd call it a bunch of tales... but I can change the text. Teeneep looks like she doesn't like it here, though, if she rolls her eyes. 

Stories aren't always collected or make sense together anyway. Genesis, for example... 

```suggestion
			`	Teeneep rolls her eyes, putting her drink down with a sharp clink. "It's superstition. Humans have their religions, the Hai have the Mythos. It's a bunch of stories about Hai who lived incredibly long lives and were involved in or responsible for incredibly influential events in our history, or more accurately pre-history for the vast majority of them. If you believe the stories literally, these Hai transcended mortal limitations and achieved immortality; our word for them translates imprecisely to either 'immortals' or 'imperishables'. When someone embodies a character from it well enough there’s a set of people who will believe it’s actually that person come back to steer things.`
```

Reply:
Would you do so if your best friend and mentor of several decades, who would not like that language, were sitting right next to you?

Reply:
I didn't know Sayari didn't like that language. She didn't say anything about the language, she only responds that more important people do care than Teeneep thinks. I don't think the reverend of my college wouldn't be offended if I called the bible a bunch of stories (certainly not the previous one anyway).

Reply:
Yeah I'm still not down with putting in "bunch" in here, but two of those commas are necessary.
```suggestion
			`	Teeneep rolls her eyes, putting her drink down with a sharp clink. "It's superstition. Humans have their religions, the Hai have the Mythos. It's a collection of stories about Hai who lived incredibly long lives and were involved in or responsible for incredibly influential events in our history, or more accurately pre-history for the vast majority of them. If you believe the stories literally, these Hai transcended mortal limitations and achieved immortality; our word for them translates imprecisely to either 'immortals' or 'imperishables'. When someone embodies a character from it well enough, there’s a set of people who will believe it’s actually that person who has come back to steer things.`
```